templates/blog/post_detail.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+
+{% block content %}
+  
+  <h1>{{ post.title }}</h1>
+  <p>Автор: {{ post.owner.username }}</p>
+  <p>Створено: {{ post.created_time|date:"d.m.Y H:i" }}</p>
+  <p>{{ post.content }}</p>
+
+  <h2>Коментарі ({{ post.comment_count }})<a style="float: right" href="{% url 'blog:create' post.id %}">+</a></h2>
```

Comment:
The URL pattern 'blog:create' might be incorrect if the intention is to create a comment for the current post. Ensure that the URL pattern matches the intended view for creating a comment and that it correctly uses the post ID.